cmd/sonobuoy/app/args.go

```diff
@@ -401,3 +395,84 @@ func AddExtractFlag(flag *bool, flags *pflag.FlagSet) {
 		"If true, extracts the results instead of just downloading the results",
 	)
 }
+
+type e2eFocusFlagType PluginEnvVars
+
+func (i *e2eFocusFlagType) String() string {
+	if i != nil && (map[string]map[string]string)(*i)["e2e"] != nil {
```

Comment:
Suggestion: make this `(map[string]map[string]string)` a named type so your type conversion is simpler

```
type FocusFlag map[string]map[string]string

func (i *e2eFocusFlagType) String() string {
	if i != nil && FocusFlag(*i)["e2e"] != nil {
```

Reply:
Its actually already a named type so that was easy to swap. For some reason I thought when indexing it I had to use the underlying map type, but you're right